tests/test_physics/test_representations.py

```diff
@@ -0,0 +1,274 @@
+
+    @pytest.mark.parametrize("scalar", [0.5, 1.2])
+    @pytest.mark.parametrize("triple", [Abc_n1, Abc_n2, Abc_n3])
+    def test_mul_with_scalar(self, scalar, triple):
+        bargmann1 = Bargmann(*triple)
+        bargmann_mul = bargmann1 * scalar
+
+        assert np.allclose(bargmann1.A, bargmann_mul.A)
+        assert np.allclose(bargmann1.b, bargmann_mul.b)
+        assert np.allclose(bargmann1.c * scalar, bargmann_mul.c)
+
+    @pytest.mark.parametrize("n", [1, 2, 3])
+    def test_mul(self, n):
+        triple1 = Abc_triple(n)
+        triple2 = Abc_triple(n)
+
+        bargmann1 = Bargmann(*triple1)
+        bargmann2 = Bargmann(*triple2)
+        bargmann_mul = bargmann1 * bargmann2
+
+        assert np.allclose(bargmann_mul.A, bargmann1.A + bargmann2.A)
+        assert np.allclose(bargmann_mul.b, bargmann1.b + bargmann2.b)
+        assert np.allclose(bargmann_mul.c, bargmann1.c * bargmann2.c)
+
+    @pytest.mark.parametrize("scalar", [0.5, 1.2])
+    @pytest.mark.parametrize("triple", [Abc_n1, Abc_n2, Abc_n3])
+    def test_div_with_scalar(self, scalar, triple):
+        bargmann1 = Bargmann(*triple)
+        bargmann_div = bargmann1 / scalar
+
+        assert np.allclose(bargmann1.A, bargmann_div.A)
+        assert np.allclose(bargmann1.b, bargmann_div.b)
+        assert np.allclose(bargmann1.c / scalar, bargmann_div.c)
+
+    @pytest.mark.parametrize("n", [1, 2, 3])
+    def test_div(self, n):
+        triple1 = Abc_triple(n)
+        triple2 = Abc_triple(n)
+
+        bargmann1 = Bargmann(*triple1)
+        bargmann2 = Bargmann(*triple2)
+        bargmann_div = bargmann1 / bargmann2
+
+        assert np.allclose(bargmann_div.A, bargmann1.A - bargmann2.A)
+        assert np.allclose(bargmann_div.b, bargmann1.b - bargmann2.b)
+        assert np.allclose(bargmann_div.c, bargmann1.c / bargmann2.c)
+
+    @pytest.mark.parametrize("n", [1, 2, 3])
+    def test_add(self, n):
+        triple1 = Abc_triple(n)
+        triple2 = Abc_triple(n)
+
+        bargmann1 = Bargmann(*triple1)
+        bargmann2 = Bargmann(*triple2)
+        bargmann_add = bargmann1 + bargmann2
+
+        assert np.allclose(bargmann_add.A, math.concat([bargmann1.A, bargmann2.A], axis=0))
+        assert np.allclose(bargmann_add.b, math.concat([bargmann1.b, bargmann2.b], axis=0))
+        assert np.allclose(bargmann_add.c, math.concat([bargmann1.c, bargmann2.c], axis=0))
+
+    @pytest.mark.parametrize("n", [1, 2, 3])
+    def test_sub(self, n):
+        triple1 = Abc_triple(n)
+        triple2 = Abc_triple(n)
+
+        bargmann1 = Bargmann(*triple1)
+        bargmann2 = Bargmann(*triple2)
+        bargmann_add = bargmann1 - bargmann2
+
+        assert np.allclose(bargmann_add.A, math.concat([bargmann1.A, bargmann2.A], axis=0))
+        assert np.allclose(bargmann_add.b, math.concat([bargmann1.b, bargmann2.b], axis=0))
+        assert np.allclose(bargmann_add.c, math.concat([bargmann1.c, -bargmann2.c], axis=0))
+
+    # def test_trace(self):
```

Comment:
@ziofil could you try running this test with `--backend=tensorflow`?
I am getting an error, and I am not sure if I'm doing something wrong or there's a small bug